gpflow/models/model.py

```diff
@@ -27,27 +37,12 @@
 from ..mean_functions import MeanFunction, Zero
 from ..utilities import ops, to_default_float
 
-InputData = tf.Tensor
-OutputData = tf.Tensor
-RegressionData = Tuple[InputData, OutputData]
 MeanAndVariance = Tuple[tf.Tensor, tf.Tensor]
 
 
-class BayesianModel(Module):
+class BayesianModel(Module, metaclass=abc.ABCMeta):
```

Comment:
you can shorten this with
```suggestion
class BayesianModel(Module, abc.ABC):
```

Reply:
just a suggestion